.github/workflows/swift-toolchain.yml

```diff
@@ -1985,6 +1985,11 @@ jobs:
           name: installer-amd64
           path: ${{ github.workspace }}/tmp
 
+      - uses: actions/download-artifact@v3
+        with:
+          name: runtime.amd64.msm
+          path: ${{ github.workspace }}/tmp
```

Comment:
Could we pick up all 3 MSMs?  (arm64, x86, and arm64)

Reply:
Sure, but I'd rather get this platform working end-to-end first.
(also, e.g., note we only upload installer-amd64.exe to the Release today)

Reply:
Right, we only upload installer-amd64, but it contains the SDK for all the platforms :)

Reply:
@fischman-bcny pointed out offline that we don't actually package the ARM64 SDK, so ...

Reply:
Per tuple, we're [not packaging the arm64](https://github.com/thebrowsercompany/swift-build/blob/b640eafa68af5eae455ad4ff13dd9298f7ab5f1b/.github/workflows/swift-toolchain.yml#L1763-L1764) SDK yet, so doesn't seem right to pick the MSM yet.
(likely both to come)